tests/capture/autograph/test_helpers.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""PyTests for AutoGraph helper functions."""
+
+from functools import WRAPPER_ASSIGNMENTS, WRAPPER_UPDATES
+from functools import wraps as ft_wraps
+
+import pytest
+
+from pennylane.capture.autograph import wraps as ag_wraps
+
+
+def test_wrapper_assignments():
+    """Test that our custom `wraps` function works the same as the functools version,
+    except for the __module__ attribute."""
+
+    # we use the custom wrapping function itself as a target
+    @ag_wraps(ag_wraps)
```

Comment:
Why is this?

Reply:
We need a target, doesn't really matter what it is, and the wraps function was already imported.

Reply:
Oh right lol. I forgot how wraps works for a sec. Resolving